src/org/zaproxy/zap/extension/formhandler/PopupMenuAddFormhandlerParam.java

```diff
@@ -0,0 +1,66 @@
+/*
+ * Zed Attack Proxy (ZAP) and its related class files.
+ *
+ * ZAP is an HTTP/HTTPS proxy for assessing web application security.
+ *
+ * Copyright 2018 The ZAP Development Team
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.zaproxy.zap.extension.formhandler;
+
+import java.awt.Component;
+
+import javax.swing.JTable;
+
+import org.parosproxy.paros.Constant;
+import org.parosproxy.paros.extension.ExtensionPopupMenuItem;
+import org.parosproxy.paros.view.View;
+import org.zaproxy.zap.extension.params.HtmlParameterStats;
+import org.zaproxy.zap.extension.params.ParamsPanel;
+import org.zaproxy.zap.extension.params.ParamsTableModel;
+
+public class PopupMenuAddFormhandlerParam extends ExtensionPopupMenuItem {
+
+	private static final long serialVersionUID = 1L;
+
+	private static PopupDialogAddField popupDialogAddField = null;

```

Comment:
This can still be moved to a local variable, we are not using it outside the method.